test/specs/WebexMeeting/MeetingPage.js

```diff
@@ -0,0 +1,26 @@
+class MeetingPage {
```

Comment:
This will be more than just the "meeting" widget page
```suggestion
class SamplesPage {
```